### ISSUE TITLE
False positive for template validations with extendedLocation property

### ISSUE THREAD

**Issue by StephenWeatherford** [microsoft/vscode-azurearmtools#1374]
```json
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {},
    "functions": [],
    "variables": {},
    "resources": [
        {
            "name": "publicIPAddress1",
            "type": "Microsoft.Network/publicIPAddresses",
            "apiVersion": "2020-11-01",
            "location": "[resourceGroup().location]",
            "tags": {
                "displayName": "publicIPAddress1"
            },
            "extendedLocation": {
                "name": "abc",
                "type": "EdgeZone"
            },
            "properties": {
                "publicIPAllocationMethod": "Dynamic",
                "dnsSettings": {
                    "domainNameLabel": "dnsname1"
                }
            }
        }
    ],
    "outputs": {}
}
```
Template validation error:
```
Template validation failed: Could not find member 'extendedLocation' on object of type 'TemplateResource'. Path 'resources[0].extendedLocation', line 16, position 31.
```

**Comment by StephenWeatherford** [comment]
Blocked because we need changes in the validation assembly

**Comment by junczhan** [comment]
@StephenWeatherford Is the issue fixed?

**Comment by StephenWeatherford** [comment]
Unfortunately not yet, it's owned by another team and so is complicated.

**Comment by StephenWeatherford** [comment]
We need upvotes from anyone who's affected by this in order to push getting this fixed.

**Comment by JunchenZ** [comment]
@StephenWeatherford  Azure Portal's new Virtual Network Create experience will be affected (the work is in progress and expected to be done & published soon)

**Comment by JunchenZ** [comment]
no update for two months

**Comment by JunchenZ** [comment]
could anyone have an update on the issue as it is blocking us from doing template deployment on vs code?

**Comment by alex-frankel** [comment]
Is this failing when the template is actually deployed, or only throwing an error in the ARM Tools VS Code extension? If it is the former, then @junchenz can you open a support ticket? If it's the latter, it shouldn't be affecting the portal deployment experience.

**Comment by alex-frankel** [comment]
Have you attempted to deploy this template in the portal? Were you blocked there? I think this is a legitimate issue and we should keep it open, but want to confirm that you are not blocked.

**Comment by JunchenZ** [comment]
i was using the txt file format to avoid the vs code error, but when i test it, arm gave me error saying the localhost uri is not accepted, it need a public accessible uri, i don't know how to get a public uri of my file which is added to my repo

**Comment by alex-frankel** [comment]
That seems like a separate error from the `extendedLocation` property. Can you share more details of what you are deploying? Is it possible to share the code, the error message, and the repro steps in depth?

**Comment by StephenWeatherford** [comment]
(And preferably in a new issue unless it's actually related to the extendedLocation false positive, thanks.)

**Comment by StephenWeatherford** [comment]
Blocked by https://github.com/microsoft/vscode-azurearmtools/issues/1458

**Comment by JunchenZ** [comment]
@alex-frankel it's may be unrelated to this, could ignore
